lib/metadata/hfbMetadata.hpp

```diff
@@ -2,14 +2,11 @@
 #define HFB_METADATA
 
 #include "buffer.h"
+#include "dataset.hpp" // for dset_id_t
 #include "metadata.h"
 
 #include <sys/time.h>
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 #pragma pack()
 
 struct hfbMetadata {
```

Comment:
Probably should uppercase this for consistency with everything else, e.g. `HFBMetadata`